client/navigation/components/container/index.js

```diff
@@ -131,79 +136,110 @@ const Container = ( { menuItems } ) => {
 						setActiveLevel( ...args );
 					} }
 				>
-					{ isRootBackVisible && (
-						<NavigationBackButton
-							className="woocommerce-navigation__back-to-dashboard"
-							href={ rootBackUrl }
-							backButtonLabel={ rootBackLabel }
-							onClick={ () => trackBackClick( 'woocommerce' ) }
-						></NavigationBackButton>
-					) }
 					{ categories.map( ( category ) => {
 						const {
 							primary: primaryItems,
 							secondary: secondaryItems,
 							plugins: pluginItems,
 						} = categorizedItems[ category.id ] || {};
 						return (
-							<NavigationMenu
-								key={ category.id }
-								title={ category.title }
-								menu={ category.id }
-								parentMenu={ category.parent }
-								backButtonLabel={
-									category.backButtonLabel || null
-								}
-								onBackButtonClick={
-									isRootBackVisible
-										? null
-										: () => trackBackClick( category.id )
-								}
-							>
-								{ !! primaryItems && (
-									<NavigationGroup>
-										{ primaryItems.map( ( item ) => (
-											<Item
-												key={ item.id }
-												item={ item }
-											/>
-										) ) }
-									</NavigationGroup>
-								) }
-								{ !! pluginItems && (
-									<NavigationGroup
-										title={
-											category.id === 'woocommerce'
-												? __(
-														'Extensions',
-														'woocommerce-admin'
-												  )
-												: null
+							<>
```

Comment:
@joshuatf I still see the duplicate key error that @psealock saw. I believe it's because the key should technically be on this `<>` fragment element.
Instead of using the fragment you can also return an array, each condition as a separate item (basically just replace `<>` with `[`. Anyway that fixes it on my end.

Reply:
Hey @louwie17 - I did not see a duplicate key error, but I did see a separate error for the missing key that you mentioned here.  Updated this to use an array in https://github.com/woocommerce/woocommerce-admin/pull/6253/commits/bdcb03b25e46d3f53789e515f2dd6c073ac2d992.

Reply:
My bad, I guess I was referring to the missing key error. Thanks for updating this :)

Reply:
No problem- just making sure we're looking at the same error.  Thanks for your help @louwie17!